### PR TITLE
Add PInvoke/NativeCallManagedComVisible tests

### PR DESCRIPTION
**Pull request by jiangzeng**

### CODE REVIEW

**Comment by jkoritzinsky** [comment]
@AaronRobinsonMSFT PTAL

**Comment by jkoritzinsky** [comment]
@dotnet-bot test OSX10.12 x64 Checked Innerloop Build and Test 
@dotnet-bot test Ubuntu arm Cross Checked Innerloop Build and Test
@dotnet-bot test Ubuntu arm Cross Checked crossgen_comparison Build and Test

File: tests/src/Interop/PInvoke/NativeCallManagedComVisible/Helpers.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace NativeCallManagedComVisible
+{
+    public static class Helpers
+    {
+        public static int S_OK = 1;
```
Comment (AaronRobinsonMSFT) [inline tests/src/Interop/PInvoke/NativeCallManagedComVisible/Helpers.cs]: Uh. `S_OK` is `0`. `S_FALSE` is `1`.

**Review by AaronRobinsonMSFT — APPROVED** [review]

**Comment by jkoritzinsky** [comment]
@dotnet-bot test OSX10.12 x64 Checked Innerloop Build and Test
@dotnet-bot test Ubuntu arm Cross Checked Innerloop Build and Test
@dotnet-bot test Ubuntu arm Cross Checked crossgen_comparison Build and Test

**Comment by jkoritzinsky** [comment]
@dotnet-bot test OSX10.12 x64 Checked Innerloop Build and Test
@dotnet-bot test Ubuntu arm Cross Checked Innerloop Build and Test
@dotnet-bot test Ubuntu arm Cross Checked crossgen_comparison Build and Test

**Comment by jkoritzinsky** [comment]
@dotnet-bot test this please

**Pull request merged**